pkg/fileutil/io.go

```diff
@@ -15,7 +17,10 @@ const (
 )
 
 var (
-	ErrNotFile = errors.New("path is not a file")
+	ErrNotFile     = errors.New("path is not a file")
```

Comment:
This is confusing: a path is not a file, although it is often filename.

Reply:
in the context of the fileutils package this should not be too confusing. Do you have an alternative phrasing that you think is more clear?